sqliteanalyzer/storageanalyzer.py

```diff
@@ -240,8 +240,10 @@ def tables(self) -> [str]:
                                             FROM sqlite_master
                                             WHERE rootpage>0
                                               AND type == "table"''')
-
-        return [t['name'] for t in tables] + ['sqlite_master']
+        # Why include sqlite_master if it doesn't hold usage data?
+        # -JL
+        #return [t['name'] for t in tables] + ['sqlite_master']
```

Comment:
Good catch! Thanks for opening this PR! I'm sorry for taking this long to review it, I somehow never received an email notification for it. :(

Just a nit: 
```suggestion
        # Do not include `sqlite_master` because it doesn't hold usage data
```